<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCaseDialog</class>
 <widget class="QDialog" name="NewCaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start a New ICESym Case</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/fromHelyx/newicons/icon.png</normaloff>:/fromHelyx/newicons/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
       <underline>true</underline>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Case Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="case_name">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="open_toolButton">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Open </string>
     </property>
     <property name="icon">
      <iconset resource="../images/images.qrc">
       <normaloff>:/fromHelyx/fromHelyx/folder-open16.png</normaloff>:/fromHelyx/fromHelyx/folder-open16.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="wizard_groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Wizard</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QComboBox" name="type_ig">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Spark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Compression</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="Bore">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>80.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="ncyls">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="nstroke">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Stroke [mm]:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="crank_radius">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>100.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="rod_length">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>125.0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Constant Tank Volume:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="vol_constant">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Con-rod Lenght [mm]:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Bore [mm]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Type of Ignition:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Compression Ratio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Number of Cylinders:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="Vol_clearance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>10.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Strokes per Cycle:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Type of Cylinder:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="default_cylinder">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="accept_pushButton">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../images/images.qrc">
         <normaloff>:/fromHelyx/fromHelyx/tick16.png</normaloff>:/fromHelyx/fromHelyx/tick16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Close</string>
       </property>
       <property name="icon">
        <iconset resource="../images/images.qrc">
         <normaloff>:/fromHelyx/fromHelyx/win_closeAll16.png</normaloff>:/fromHelyx/fromHelyx/win_closeAll16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="blank_case_radioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Blank Case</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>180</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QRadioButton" name="wizard_radioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start From Wizard</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="new_case_checkBox">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start a New Case</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>new_case_checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>check_new_case()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wizard_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>enable_wizard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_toolButton</sender>
   <signal>released()</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>open_case()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept_pushButton</sender>
   <signal>released()</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>accept_d()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_pushButton</sender>
   <signal>released()</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>default_cylinder</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NewCaseDialog</receiver>
   <slot>use_default_cylinder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_new_case()</slot>
  <slot>enable_wizard()</slot>
  <slot>open_case()</slot>
  <slot>accept_d()</slot>
  <slot>use_default_cylinder()</slot>
 </slots>
</ui>
